ts/store/reducers/entities/services/servicePreference.ts

```diff
@@ -0,0 +1,62 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import { getType } from "typesafe-actions";
+import { createSelector } from "reselect";
+import { fromNullable } from "fp-ts/lib/Option";
+import { Action } from "../../../actions/types";
+import {
+  loadServicePreference,
+  upsertServicePreference
+} from "../../../actions/services/servicePreference";
+import { ServicePreferenceResponse } from "../../../../types/services/ServicePreferenceResponse";
+import { getNetworkErrorMessage } from "../../../../utils/errors";
+import { IndexedById } from "../../../helpers/indexer";
+import { toError, toLoading, toSome, toUpdating } from "../../IndexedByIdPot";
+import { GlobalState } from "../../types";
+
+export type ServicePreferenceState = IndexedById<
```

Comment:
Do we need to store the preference of multiple services?
Because if we need this info only when a service detail is displayed we could have only the single details instead of a list, e.g `currentServicePreference`

Remember that this section of the store (`entities`) is persisted so there is no deletion logic it will increase without any limits